kebab_city_tour/src/Map.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './index.css';
+import Header from './Header';
+import { MapContainer, TileLayer } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+
+export default function Map() {
+  const legnicaBounds = [
+    [51.165803, 16.148615],
+    [51.232924, 16.177626],
+    [51.202710, 16.245260],
+  ];
+
+  return (
+    <div className="min-h-screen bg-gray-100">
+      <Header />
+      <main className="flex h-[calc(100vh-6rem)] px-4 py-2">
+        <div className="bg-gray-300 w-[73%] h-full mr-2 flex items-center justify-center">
+          <MapContainer 
+            bounds={legnicaBounds} 
+            className="w-full h-full rounded-lg shadow-lg"
+            scrollWheelZoom={true}
+            maxBounds={legnicaBounds}
+            maxBoundsViscosity={1.0}
+            minZoom={13}
+          >
+            <TileLayer
+              url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+              attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+            />
+          </MapContainer>
+        </div>
+        <div className="w-[24%] h-full flex items-center justify-center text-center">
+          <p className="text-xl">LISTA / TABELA</p>
+        </div>
```

Comment:
The PR should be about main page but if your really want to add this then why don't you create actual list or drawer

Reply:
#15